src/pipelines.py

```diff
@@ -345,117 +226,79 @@ def mask_postprocessing(model, config, save_output=False):
                                 },
                        cache_dirpath=config.env.cache_dirpath,
                        save_output=save_output)
 
-    return binary_fill
-
-
-def watershed_contours(mask, contour, config, save_output=True):
-    watershed_contour = Step(name='watershed_contour',
-                             transformer=WatershedContour(),
-                             input_steps=[mask, contour],
-                             adapter={'images': ([(mask.name, 'binarized_images')]),
-                                      'contours': ([(contour.name, 'binarized_images')]),
+    category_mapper = Step(name='category_mapper_borders',
+                           transformer=make_apply_transformer(categorize_image,
+                                                              output_name='categorized_images'),
+                           input_steps=[mask_resize],
+                           adapter={'images': ([('mask_resize_borders', 'resized_images')]),
                                       },
-                             cache_dirpath=config.env.cache_dirpath,
-                             save_output=save_output)
-
-    drop_smaller = Step(name='drop_smaller',
-                        transformer=Dropper(**config.dropper),
-                        input_steps=[watershed_contour],
-                        adapter={'labels': ([('watershed_contour', 'detached_images')]),
-                                 },
-
-                        cache_dirpath=config.env.cache_dirpath,
-                        save_output=save_output)
-    return drop_smaller
+                           cache_dirpath=config.env.cache_dirpath,
+                           save_output=save_output)
 
+    borders_filter = Step(name='borders_filter_borders',
+                          transformer=make_apply_transformer(partial(get_channel,
+                                                             channel=config.postprocessor.channels.index('borders')),
+                                                             output_name='nuclei_images'),
+                          input_steps=[category_mapper],
+                          adapter={'images': ([('category_mapper_borders', 'categorized_images')]),
+                                   },
+                          cache_dirpath=config.env.cache_dirpath,
+                          save_output=save_output)
 
-def nuclei_labeler(postprocessed_mask, config, save_output=True):
-    labeler = Step(name='labeler',
-                   transformer=NucleiLabeler(),
-                   input_steps=[postprocessed_mask],
-                   adapter={'images': ([(postprocessed_mask.name, 'binarized_images')]),
-                            },
-                   cache_dirpath=config.env.cache_dirpath,
-                   save_output=save_output)
-    return labeler
+    return borders_filter
 
 
 PIPELINES = {'unet': {'train': partial(unet, train_mode=True),
```

Comment:
@taraspiotr I think we should have unet_masks and unet_masks_with_borders pipelines for training (as I believe you suggested). It would be easier for people to reproduce it easily